packages/datadog-instrumentations/src/mocha.js

```diff
@@ -0,0 +1,91 @@
+const { addHook, channel, AsyncResource } = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+const runTestStartCh = channel('ci:mocha:run-test:start')
+const errorCh = channel('ci:mocha:run-test:error')
+const skipCh = channel('ci:mocha:run-test:skip')
+const runTestEndCh = channel('ci:mocha:run-test:end')
+const runTestEndAsyncCh = channel('ci:mocha:run-test:async-end')
+const runTestsEndCh = channel('ci:mocha:run-tests:end')
+const hookErrorCh = channel('ci:mocha:hook-error')
+const mochaEachCh = channel('ci:mocha:mocha-each')
```

Comment:
What is the actual even here? Looks like `mocha-each` is the library and not the operation.

Reply:
I'll change it to `ci:mocha:test:parameterize`